extensions/sql-migration/src/dialog/tdeConfiguration/tdeConfigurationDialog.ts

```diff
@@ -0,0 +1,331 @@
+
+		manualMethodContainer.addItems([
+			manualMethodButton,
+			this._manualMethodWarningContainer
+		]);
+
+		radioButtonContainer.addItems([
+			adsMethodContainer,
+			manualMethodContainer]);
+
+		const container = _view.modelBuilder.flexContainer()
+			.withLayout({ flexFlow: 'column' })
+			.withItems([
+				chooseMethodText,
+				radioButtonContainer
+			])
+			.component();
+
+		return container;
+	}
+
+	private createAdsConfirmationContainer(_view: azdata.ModelView): azdata.FlexContainer {
+		const container = _view.modelBuilder.flexContainer().withProps({
+			CSSStyles: {
+				'flex-direction': 'column'
+			}
+		}).component();
+
+		const adsMethodInfoMessage = _view.modelBuilder.infoBox()
+			.withProps({
+				text: constants.TDE_WIZARD_ADS_CERTS_INFO,
+				style: 'information',
+				CSSStyles: {
+					...styles.BODY_CSS,
+					'margin': '4px 14px 0px 14px'
+				}
+			}).component();
+
+		const networkPathLabel = _view.modelBuilder.text()
+			.withProps({
+				value: constants.TDE_WIZARD_CERTS_NETWORK_SHARE_LABEL,
+				description: constants.TDE_WIZARD_CERTS_NETWORK_SHARE_INFO,
+				requiredIndicator: true,
+				CSSStyles: {
+					...styles.LABEL_CSS,
+					'margin': '4px 0 14px 45px'
+				}
+			}).component();
+		this._networkPathText = _view.modelBuilder.inputBox()
+			.withProps({
+				value: '',
+				width: '300px',
+				placeHolder: constants.TDE_WIZARD_CERTS_NETWORK_SHARE_PLACEHOLDER,
+				required: true,
+				CSSStyles: { ...styles.BODY_CSS, 'margin-top': '-1em', 'margin-left': '45px' }
+			}).component();
+
+		this._adsConfirmationCheckBox = _view.modelBuilder.checkBox()
+			.withProps({
+				label: constants.TDE_WIZARD_MIGRATION_OPTION_ADS_CONFIRM,
+				checked: this.migrationStateModel.tdeMigrationConfig.isTdeMigrationMethodAdsConfirmed(),
+				CSSStyles: {
+					...styles.BODY_CSS,
+					'margin': '10px 0 14px 15px'
+				}
+			}).component();
+		this._disposables.push(
+			this._adsConfirmationCheckBox.onChanged(async checked => {
+				this.migrationStateModel.tdeMigrationConfig.setAdsConfirmation(
+					checked,
+					this._networkPathText.value ?? '');
+				await this.updateUI();
+			}));
+
+		container.addItems([
+			adsMethodInfoMessage,
+			networkPathLabel,
+			this._networkPathText,
+			this._adsConfirmationCheckBox]);
+
+		return container;
+	}
+
+	private createManualWarningContainer(_view: azdata.ModelView): azdata.FlexContainer {
+		const container = _view.modelBuilder.flexContainer().withProps({
+			CSSStyles: {
+				'flex-direction': 'column'
+			}
+		}).component();
+
+		const manualMethodConfirmationDialog = _view.modelBuilder.infoBox()
+			.withProps({
+				text: constants.TDE_WIZARD_MIGRATION_OPTION_MANUAL_WARNING,
+				style: 'warning',
+				CSSStyles: {
+					...styles.BODY_CSS,
+					'margin': '4px 14px 0px 14px'
+				},
+				links: [{
+					text: constants.LEARN_MORE,
+					url: 'https://learn.microsoft.com/en-us/azure/azure-sql/managed-instance/tde-certificate-migrate',
```

Comment:
please remove the "/en-us/" from this and any other hardcoded URLs, as this will force the user (even if non-English) onto the American English docs page

Reply:
Fixed